### PR TITLE
Notification after theme reset, warning about images regeneration

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | i think that we needed that information as after adding some image types i forgot to regenerate them :D
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | n/a
| How to test?  | reset theme to defaults

### CODE REVIEW

**Comment by Shudrum** [comment]
Thank you again @kpodemski!

Ping @AlexEven for wording.

File: controllers/admin/AdminThemesController.php [hunk 1]
```diff
@@ -279,7 +279,13 @@ public function postProcess()
             $this->theme_manager->uninstall(Tools::getValue('theme_name'));
             $this->redirect_after = $this->context->link->getAdminLink('AdminThemes');
         } elseif (Tools::getValue('action') == 'resetToDefaults') {
-            $this->theme_manager->reset(Tools::getValue('theme_name'));
+            if ($this->theme_manager->reset(Tools::getValue('theme_name'))) {
+                $this->confirmations[] = $this->trans(
+                    'Your theme has been correctly reseted to default settings, you may want to regenerate your images again.',
```
Comment (xBorderie) [inline controllers/admin/AdminThemesController.php]: "reseted" does not exist, you should use "has been correctly reset" :)
http://www.verbix.com/webverbix/English/reset.html

Comment (AlexEven) [inline controllers/admin/AdminThemesController.php]: +1

Comment (AlexEven) [inline controllers/admin/AdminThemesController.php]: Yes, indeed! Thank you!

**Review by AlexEven — COMMENTED** [review]
+1 for @xBorderie's comment

**Comment by kpodemski** [comment]
Thanks guys, i messed up this PR, i've created another one here:
https://github.com/PrestaShop/PrestaShop/pull/7570

**Comment by xBorderie** [comment]
I think you could have saved it [with a simple git squash](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/). Thanks for recreating the PR!

**Pull request closed without merging**